src/client/stylesheets/style.less

```diff
@@ -537,20 +572,24 @@ hr.wide {
 }
 
 .label-checkbox {
-	padding: 0 .5em;
+	padding: 0 0.5em;
 	vertical-align: text-top;
 }
 
 .addToCollectionModal-body {
 	margin-right: 1em;
 	margin-left: 1em;
 	height: 18em;
-	overflow-y:scroll;
+	overflow-y: scroll;
 }
 
 .checkboxes {
-	margin-right: 0.5em !important;;
+	margin-right: 0.5em !important;
 	vertical-align: text-top;
 }
 
-
+.custom-width {
```

Comment:
is it used at many places ? 
or if it is used at some specific location for eg. footer then the className must be more intuitive

Reply:
thanks for the review, I agree with you on this.